### DISCUSSION TITLE
Snipe-IT LXC would be great!

### DISCUSSION THREAD

**Discussion by SenseiLeCrook** [tteck/Proxmox#3393] [Ideas]
A quick deploy of the IT asset management software https://snipeitapp.com/ would be fantastic.

**Comment by ThePaulMichaels** [comment]
I'd also LOVE to see this!

**Comment by hgelpke** [comment]
I actually came here today to see if this existed after learning about it.

**Comment by eitch** [comment]
I too came to see if this was available.

**Comment by moayadalmalat** [comment]
+1
